packages/fxa-metrics-processor/src/lib/amplitude/transforms/user-properties.ts

```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Event, EventContext, Device } from './types';
+import { toSnakeCase } from './common';
+import { ParsedUserAgentProperties } from '../../../../../fxa-shared/metrics/user-agent';
+import { mapBrowser, BrowserProps } from './user-agent';
+
+const DAY = 1000 * 60 * 60 * 24;
+const WEEK = DAY * 7;
+const FOUR_WEEKS = WEEK * 4;
+type DevicePeriod = typeof DAY | typeof WEEK | typeof FOUR_WEEKS;
```

Comment:
All of these resolve to 'number', is that useful to split them out like this when it ends up being a number? Do we sometimes get this value as a DAY, WEEK, or FOUR_WEEKS? If not maybe just calling this a 'number' is fine.